source/templates.tex

```diff
@@ -6044,8 +5936,8 @@
 \pnum
 An implementation shall not implicitly instantiate a function template,
 a variable template,
-a member template, a non-virtual member function, a member class, a
-static data member of a class template, or a substatement of a constexpr if
+a member template, a non-virtual member function, or a member class or
+static data member of a templated class, or a substatement of a constexpr if
```

Comment:
there are 2 `, or`s in this sentence - is the 1st newly added one a mistake? i.e. "function, or a member class" -> "function, a member class"

if not, either add `;`s to clarify the intended grouping or employ bullets

Reply:
fixed